plugins/oauth/web_client/stylesheets/login.styl

```diff
@@ -29,15 +31,33 @@
     border-right 1px solid white
   .g-oauth-button-text
     display inline-block
-    padding 0 20px
+    text-align center
+    width 100px
```

Comment:
I selected the ``width 160px`` for the whole button with the largest string of ``StackExchange`` in mind. Once that provider is added, we may need to expand this ``width 100px`` for the text as well. If it looks good, it's fine for now though.